playground/backend/internal/fs_tool/fs.go

```diff
@@ -109,6 +111,14 @@ func (l *LifeCycle) GetAbsoluteExecutableFilePath() string {
 	return absoluteFilePath
 }
 
+// GetAbsoluteCompiledFilePath returns absolute filepath to compiled file (/path/to/workingDir/executable_files/{pipelineId}/bin/{pipelineId}.class for java SDK).
+func (l *LifeCycle) GetAbsoluteCompiledFilePath() string {
```

Comment:
what is the "compiled" here? I assume that here it is a compiled go file, but it is not clear